assets/js/custom/listing.js

```diff
@@ -26,19 +26,18 @@ $('.listing').each(function(){
 
     // If one or more items are checked, toggle the visibility of the listing's toolbar
     if ($(toolbar).length) {
-        $(select_checkboxes).on('change', function(e){
-            select_checkboxes_checked = $(context).find('.listing-item-select input[type="checkbox"]:checked');
-
-            if ($(select_checkboxes_checked).length) {
-                $(context).addClass('toolbar-active');
-                $('.button-fab').prop('disabled', true).addClass('disabled');
-            } else {
-                $(context).removeClass('toolbar-active');
-                $('.button-fab').prop('disabled', false).removeClass('disabled');
-            }
-
+        $('#bulk-action').on('click', function(e){
+            $(context).addClass('toolbar-active');
+            $('.button-fab').prop('disabled', true).addClass('disabled');
             $(toolbar).find('.listing-toolbar-summary .total').text($(select_checkboxes_checked).length);
         });
+
+         $('#bulk-close').on('click', function(e){
+            $(context).removeClass('toolbar-active');
+            $('.button-fab').prop('disabled', false).removeClass('disabled');
+        });
+
+        
```

Comment:
clicking "Bulk Actions" causes a page refresh. Prevent default on `<a>` maybe?

Reply:
This isn't happening for me. There is no <a> tag, so nothing to prevent default on.